streaming_jupyter_integrations/schema_view.py

```diff
@@ -58,8 +60,9 @@ def get_schema(self) -> SchemaRoot:
         current_catalog_name = self.st_env.execute_sql("SHOW CURRENT CATALOG").collect().next()[0]
         current_database_name = self.st_env.execute_sql("SHOW CURRENT DATABASE").collect().next()[0]
 
-        catalogs = self.st_env.execute_sql("SHOW CATALOGS").collect()
-        catalogs_tree = [self._build_catalog(catalog) for catalog in catalogs]
+        with ThreadPoolExecutor(self.parallelism) as executor:
+            catalogs = self.st_env.execute_sql("SHOW CATALOGS").collect()
+            catalogs_tree = [executor.submit(self._build_catalog, catalog) for catalog in catalogs]
```

Comment:
Do not do this. It makes no sense. Only the lowest level should be parallelised ("build_tables").